core/appeal/service.go

```diff
@@ -850,14 +851,24 @@ func (s *Service) getPoliciesMap(ctx context.Context) (map[string]map[uint]*doma
 	if err != nil {
 		return nil, err
 	}
+
+	// make sure to sort policies with latest version desc
+	sort.Slice(policies, func(i, j int) bool {
+		return policies[i].Version > policies[j].Version
+	})
+
 	policiesMap := map[string]map[uint]*domain.Policy{}
-	for _, p := range policies {
+	for i, p := range policies {
 		id := p.ID
 		version := p.Version
 		if policiesMap[id] == nil {
 			policiesMap[id] = map[uint]*domain.Policy{}
 		}
 		policiesMap[id][version] = p
+		if i == 0 {
+			// set the map 0 as the latest version
+			policiesMap[id][0] = p
+		}
```

Comment:
please modify the logic under the assumption that policy IDs also can be multiple. 
Example: (`{policy_id}@{version}`)
- `test-policy@1`
- `test-policy@2`
- `other-policy@1`
- `other-policy@2`
- `other-policy@3`

Reply:
@rahmatrhd updated the flow based on the cases above ya